courses/machine_learning/deepdive2/text_classification/labs/reusable_embeddings.ipynb

```diff
@@ -16,7 +16,7 @@
     "## Introduction\n",
```

Comment:
<p>remove TODOs and Tasks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/863/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
https://github.com/GoogleCloudPlatform/training-data-analyst/pull/863/commits/75686e169e7ba64ac685d80d3e83eb6d4fe11c2d